metrics/README.md

```diff
@@ -25,3 +25,69 @@ function in that file has the ability to also `curl` or `socat` the JSON results
 by environment variables (see the file source for details). This method has been used to store results in
 Elasticsearch and InfluxDB databases for instance, but should be adaptable to use with any REST API that accepts
 JSON input.
+
+## Scaling execution
+This section describes a complete step-by-step scaling execution upto results reporting by using `scaling/k8s_scale.sh` tool which launch a series of workloads and take memory metric measurements after each launch.
+
```

Comment:
... tool which **launches** a series of ....

Reply:
Good catch!